static/app/components/events/interfaces/debugMeta-v2/debugImageDetails/index.tsx

```diff
@@ -397,28 +397,19 @@ const StyledButtonBar = styled(ButtonBar)`
 `;
 
 export const modalCss = css`
-  .modal-content {
+  [role='document'] {
```

Comment:
`document`?

Reply:
Yeah this is what was used before as well.